### PR TITLE
Jav 315 response out

### PR DESCRIPTION
**Pull request by seanyinx**

### CODE REVIEW

File: saga-core/src/main/java/io/servicecomb/saga/core/CompositeSagaResponse.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package io.servicecomb.saga.core;
```
Comment (WillemJiang) [inline saga-core/src/main/java/io/servicecomb/saga/core/CompositeSagaResponse.java]: Please add the license header for the new added file.

File: saga-core/src/main/java/io/servicecomb/saga/core/PersistentLog.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package io.servicecomb.saga.core;
```
Comment (WillemJiang) [inline saga-core/src/main/java/io/servicecomb/saga/core/PersistentLog.java]: license header.

File: saga-core/src/main/java/io/servicecomb/saga/core/SagaContext.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package io.servicecomb.saga.core;
```
Comment (WillemJiang) [inline saga-core/src/main/java/io/servicecomb/saga/core/SagaContext.java]: license header.

File: saga-core/src/main/java/io/servicecomb/saga/core/CompositeSagaResponse.java (hunk 1 above)
Comment (seanyinx) [reply saga-core/src/main/java/io/servicecomb/saga/core/CompositeSagaResponse.java]: done

File: saga-core/src/main/java/io/servicecomb/saga/core/PersistentLog.java (hunk 2 above)
Comment (seanyinx) [reply saga-core/src/main/java/io/servicecomb/saga/core/PersistentLog.java]: done

File: saga-core/src/main/java/io/servicecomb/saga/core/SagaContext.java (hunk 3 above)
Comment (seanyinx) [reply saga-core/src/main/java/io/servicecomb/saga/core/SagaContext.java]: done

Comment (seanyinx) [reply saga-core/src/main/java/io/servicecomb/saga/core/SagaContext.java]: done

**Pull request merged**